FreeRTOS/Demo/CORTEX_MPS2_QEMU_IAR_GCC/Readme.md

```diff
@@ -0,0 +1,8 @@
+# Running with VSCode
+
+1. Ensure [C/C++ extension](https://marketplace.visualstudio.com/items?itemName=ms-vscode.cpptools) is installed in VSCode.
```

Comment:
Would it be better to define these as prerequisites:
1. Install C/C++ VS Code extension.
2. Install arm-none-eabi-gcc.
3. Install make. 
4. Instructions to check if the required binaries are in PATH.